db/re/item_db.txt

```diff
@@ -1165,6 +1165,7 @@
 1832,Velum_Claw,Vellum Claw,5,20,,650,160,,1,0,0x00008100,63,2,2,4,95,1,12,{ bonus2 bAddRace,RC_Player,100; .@r = getrefine(); if(.@r>7) { bonus2 bAddRace,RC_Player,20; } if(.@r>9) { bonus2 bVariableCastrate,"MO_EXTREMITYFIST",-100; bonus2 bFixedCastrate,"MO_EXTREMITYFIST",-100; bonus4 bAutoSpellOnSkill,"MO_EXPLOSIONSPIRITS","CH_SOULCOLLECT",1,10000; bonus2 bHPLossRate,500,3000; } },{},{}
 1833,Claw_Of_Flash,Claw of Flash,5,60000,,400,140,,1,2,0x00008100,63,2,2,4,105,1,12,{ bonus bAspd,1; bonus bAspdRate,getrefine(); bonus2 bAddSize,Size_Medium,getrefine(); },{},{}
 1834,TE_Woe_Fist,TE Woe Fist,5,0,,0,150,,1,0,0x00008100,63,2,2,3,40,1,12,{ bonus2 bAddRace,RC_Player,40; bonus2 bAddEff,Eff_Freeze,3000; },{},{}
+1835,Spartacus,Spartacus,5,0,,600,100,,1,0,0x00008100,63,2,2,4,50,2,12,{ bonus bUnbreakableWeapon,1; .@r = getrefine(); bonus bAtkRate,.@r*1; if(.@r>9) { bonus bNoSizeFix,0; } },{},{}
```

Comment:
The `bAtkRate` bonus should be `bonus2 bAddClass,Class_All,.@r;`

Reply:
@aleos89 I made the requested changes using GitHub Desktop and Atom, but I don't know how to proceed... Could you teach me?

![git](https://user-images.githubusercontent.com/34192842/35697286-da441416-0770-11e8-8492-feff353139cf.png)

![git2](https://user-images.githubusercontent.com/34192842/35697296-e0385e2c-0770-11e8-891c-892f854c7431.png)

Reply:
Slot [2]

Reply:
Thank you zack!
I did it. ^_^